src/Tribe/Aggregator/Record/Queue_Realtime.php

```diff
@@ -196,6 +210,59 @@ public function get_unable_to_continue_processing_data() {
 	 * @return mixed|string|void
 	 */
 	public function get_progress_message_data( $queue, $percentage, $done ) {
+		_deprecated_function( __METHOD__, 'TBD' );
+
+		return wp_json_encode( $this->get_progress_data($queue, $percentage, $done) );
+	}
+
+	/**
+	 * Get the data that is used to construct the current status of the EA progress bar.
+	 *
+	 * @since TBD
+	 *
+	 * @return array<string, mixed> An array with the details of the progress bar.
+	 */
+	private function get_queue_progress_data() {
+		if ( $this->record_id <= 0 ) {
+			return [];
+		}
+
+		// Load the queue
```

Comment:
:no_entry_sign: Inline comments must end in full-stops, exclamation marks, or question marks

<sub>hash: d0e730f247632c6910724a444d7e9ab3</sub>